core/src/main/java/com/adobe/aem/commons/assetshare/content/impl/MetadataSchemaPropertiesImpl.java

```diff
@@ -41,7 +41,8 @@
 
     private static final String NN_FIELD = "field";
 
-    private static final String[] RT_FIELDS = { "granite/ui/components/foundation/form/field", "granite/ui/components/coral/foundation/form/field" };
```

Comment:
@arpithaar sorry to be a bother. Do you mind creating an issue for the Metadata schema change and then submiting this class as a separate PR? It will help us for tracking purposes...